db/migrate/20150818181427_update_tenant_divisible_on_existing_rows.rb

```diff
@@ -1,7 +1,8 @@
 class UpdateTenantDivisibleOnExistingRows < ActiveRecord::Migration
   class Tenant < ActiveRecord::Base
   end
```

Comment:
Every stub, yes.  Migration stubs are a snapshot in time, and at this present time, Tenant does not have STI.  If you don't put it in the disabling, this PR has a high probability of failing in the future if STI ever gets added.  It has happened a number of times to us in the past, and it's a pain in the neck to try and figure out when it does.